generator/generate-library-list.sh

```diff
@@ -60,36 +61,40 @@ for d in ./google-cloud-java/*java-*/; do
     echo "$d: release_level: $release_level"
     continue
   fi
+  if [[ $artifact_id == "google-cloud-vision" ||  $artifact_id == "google-cloud-kms" ]] ; then
+    echo "$artifact_id is already present in manual modules."
+    continue
+  fi
+
+  # get monorepo-name as pattern ./google-cloud-java/<monorepo_name>/
+  monorepo_name=$(echo $d | sed 's#^./google-cloud-java/##' | sed 's#/$##')
+
+  # get folder location source of truth from ".OwlBot.yaml"
+  googleapis_path=$(grep 'java/gapic-google' $d/.OwlBot.yaml | sed 's#^.*"/##' | sed 's#/[^/]*/.\*-java.*$##')
+  repo_folder=$(grep 'java/gapic-google' $d/.OwlBot.yaml -A1| tail -n1 |  sed "s#^.*$monorepo_name/[^/]*/##"| sed 's#"$##')
+
+  # figure out path to look out changes for: v[1-9]
+  # taking the latest version that's not alpha/beta
+  version_folder=$(find "$d$repo_folder/main/java/com/google/" -type d -name 'v[0-9]' |sort -d -r | head -n 1 | sed "s#^$d##")
+  version_number=$(echo $version_folder | sed 's#.*/##')
+  googleapis_folder="$googleapis_path/$version_number"
 
-  # get version from versions.txt
+  # get commitish from git log
+  # criteria: changes happen before tag,  touches path, and with changes in googleapis/googleapis
+  cd $d || { echo "Failed to get into directory $d"; exit 1; }
+  googleapis_committish=$(git log v1.1.0 -- "$version_folder" | grep -m 1 'Source-Link:.*googleapis/googleapis.*' | sed 's#^.*/commit/##')
```

Comment:
Here I assume `v.1.1.0` is meant to be the flag argument instead?

Reply:
yes! good catch.